service/src/main.rs

```diff
@@ -16,9 +16,10 @@
 */
 
 #![cfg_attr(test, feature(assert_matches))]
+#![allow(clippy::redundant_closure_call)]
```

Comment:
Why do we have to allow it in the CI if we also allow it here? Does it occur all over the place?

Reply:
I did it because I thought it would make sense that the CI explicitly disables it so the developers know it, but wanted something that would not confuse someone just running `cargo clippy` without checking the CI scripts.

I think it is okay to leave it as is, however if you think it would be better to remove one, then I will do that.

Reply:
I see, valid perspective! I have a hard time formulating a clear opinion around it, what I clearly agree with: a dev needs to be able to call `cargo clippy` and it passes. 

After thinking a bit, I believe it is better to have it in the code only in this case, create an issue and refer to the issue here. If this is a known false positive in clippy, even better, then you can just link to that one.